portfolio/src/main/webapp/script-home.js

```diff
@@ -101,8 +103,33 @@ function makeDiv(comment){
   divOfComment.appendChild(nameOfCommenter);
   divOfComment.appendChild(dateOfComment);
   divOfComment.appendChild(commentString);
-  divOfComment.style.border='3px solid #b31b1b';
-  divOfComment.style.margin='15px 0 15px';
-  divOfComment.style.padding='10px';
+  divOfComment.className = 'comment-div'
   return divOfComment;
 }
+
+/**
+ * Returns the URL to the servlet and pdates the query
+ * string based on the max-number-display and sort-comments select.
+ */
+function updateComments() {
```

Comment:
Rename to buildUpdateCommentsURL to make it clear that this function isn't actually modifying comments and only building the request path.